### DISCUSSION TITLE
Mod doesnt appear in sandbox setting MP

### DISCUSSION THREAD

**Discussion by PeruvianMacaco** [TEHE-Studios/ExpandedHelicopterEvents#199] [Q&A]
Hi, the mod settings are not appearing for me in the sandbox settings

**Comment by Chuckleberry-Finn** [comment]
If you're using the Host menu (or single player for that matter) the mod needs to be enabled in the main menu.

**Comment by imnotazombie24** [comment]
I am having the same issue. I have a MP server through Indifferent Broccoli and I'm trying to use this mod for the first time ever. I placed it after the vehicles the first time and when I restarted the server it didn't show up in pzserver_sandboxvar so I put it at the end of my workshop/mod id's thinking it needed to be there and still the same thing. I only have it in their once. There is no duplicates.
